JavaScript2/Week5/homework.md

```diff
@@ -113,12 +113,8 @@ You can see `CommandLine` in the URL. These are called "query parameters" and le
 
 ```
 How to hand in your homework:
-• Clone your existing "hyf-javascript2" Github repository.
-• Create a new folder "week2" USING THE COMMAND LINE 
-• Save your homework files inside this folder. 
-• When you are done with your homework use add/commit and push to upload your homework. 
-• Write a description for your “commit”.
-• Your hyf-javascript2/week2 should now contain all your homework files.
-Place the link to your repository folder in Trello.
+• Create a new repository "hyf-javascript2". Also create a new folder "week2" inside this repository. 
```

Comment:
I wasn't sure if this should be week4 or if I'm maybe misunderstanding it. Should this be changed?

Reply:
It is week 5. According to the folder. Make sure it relates to week 5 @wistoft